doc/users/next_whats_new/histogram_vectorized_parameters.rst

```diff
@@ -0,0 +1,36 @@
+Vectorize ``hatch``, ``edgecolor``, ``linewidth`` and ``linestyle`` in *hist* methods
+---------------------------------------------------------------------------------------
```

Comment:
Not technically required, but let's keep a matching underline length for aestethics.
```suggestion
-------------------------------------------------------------------------------------
```